src/settings/settings.js

```diff
@@ -1,37 +1,52 @@
-//Wait for the password to be submitted 
-document.getElementById('password-form').addEventListener('submit', async function(event){
-    event.preventDefault();
-    const newPassword = document.getElementById('new-password').value;
-    const confirmPassword = document.getElementById('confirm-password').value;
-    //Check if new password and confirmed password are the same. If they are, hash the password and store it in
-    //local storage.
-    if (newPassword === confirmPassword){
-        const hashedPassword = await hashPassword(confirmPassword);
-        localStorage.setItem('storedHashedPassword', hashedPassword);
-        window.location.href = '/src/calendar/calendar.html';
-    }
-     //console log incorrect password if the passwords do not match
-     //TODO: show incorrect password error on the page
-    else{
-        const errorMessage = document.getElementById('error-message');
-        errorMessage.style.display = 'block';
-    }
-})
-//Toggles visibility of the try again button when an error message is shown.
-const tryAgainBtn = document.getElementById('try-again-button')
-tryAgainBtn.addEventListener('click', function(){
-    const errorMessage = document.querySelector('.error');
-    errorMessage.style.display='none';
-})
-//Changes the visibility of the password settings form. If the password protection option is checked, then the form is visible.
-document.getElementById('toggle-password-form').addEventListener('change', function() {
-    const form = document.getElementById('password-form');
-    if (this.checked) {
-        form.classList.remove('hidden');
-    } else {
-        form.classList.add('hidden');
-    }
-});
+import { setStatusMDE, getStatusMDE } from "../backend-storage/mde-mode-api.js";
+
```

Comment:
This needs to be removed now, roght?

Reply:
nope, we still need markdown editing optionality in settings